tests/fixture/testnet/cmd/main.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/testnet"
+	"github.com/ava-labs/avalanchego/tests/fixture/testnet/local"
+)
+
+func main() {
+	var networkDir string
+	var rootDir string
+	rootCmd := &cobra.Command{
+		Use:   "testnetctl",
+		Short: "testnetctl commands",
+	}
+	rootCmd.PersistentFlags().StringVar(&networkDir, "network-dir", os.Getenv(local.NetworkDirEnvName), "The path to the configuration directory of a local network")
+	rootCmd.PersistentFlags().StringVar(&rootDir, "root-dir", os.Getenv(local.RootDirEnvName), "The path to the root directory for local networks")
+
+	var execPath string
+	var nodeCount uint8
+	var fundedKeyCount uint8
+	var useStaticPorts bool
+	var initialStaticPort uint16
+	startNetworkCmd := &cobra.Command{
+		Use:   "start-network [/path/to/avalanchego]",
+		Short: "Start a new local network",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(execPath) == 0 {
+				return fmt.Errorf("--avalanchego-path or %s are required", local.AvalancheGoPathEnvName)
+			}
+
+			// Root dir will be defaulted on start if not provided
+
+			network := &local.LocalNetwork{
+				LocalConfig: local.LocalConfig{
+					ExecPath:          execPath,
+					UseStaticPorts:    useStaticPorts,
+					InitialStaticPort: initialStaticPort,
+				},
+			}
+			ctx, cancel := context.WithTimeout(context.Background(), local.DefaultNetworkStartTimeout)
+			defer cancel()
+			network, err := local.StartNetwork(ctx, os.Stdout, rootDir, network, int(nodeCount), int(fundedKeyCount))
+			if err != nil {
+				return err
+			}
+
+			// Symlink the new network to the 'latest' network to simplify usage
+			networkRootDir := filepath.Dir(network.Dir)
+			networkDirName := filepath.Base(network.Dir)
+			latestSymlinkPath := filepath.Join(networkRootDir, "latest")
+			if err := os.Remove(latestSymlinkPath); err != nil && !os.IsNotExist(err) {
```

Comment:
`os.IsNotExist` is discouraged.

Looking around - seems like there are a few places to cleanup: `os.Is[a-zA-Z]+\(`